damage_calculator.py

```diff
@@ -180,9 +181,9 @@ def calc_dps_stats(client, god, build, baseAttSpeed):
     for item in build:
         stat = get_special_item(item)
         if "Physical Armor Reduction" in stat.keys() and item == "The Executioner":
-            armor_reduction_per = 7
+            armor_reduction_per = 7 # dont think this is needed as 295 takes exe into account?
```

Comment:
yea i think you can take and _item == "The Executioner"_ out of the if statement